test/surface_test.py

```diff
@@ -2283,6 +2283,34 @@ def test_blit_blend_rgba(self):
                     % (dst.get_at((x, y)), tst.get_at((x, y)), x, y),
                 )
 
+    def test_blit_blend_premultiplied(self):
```

Comment:
A couple of ideas to improve this test.

1. what is the formula it is supposed to hold to? Then the python formula can be easily compared to the other one.
2. edge cases for color could be used. 0, 2, 9, 127, 128, 200, 255.
3. edge case surfaces could be called. 0, 4, 9, 16, 17, 67, 69

Reply:
I have added tests to cover theses cases and a couple of others with surface formats (alpha on or off).